File: Atividade 2/sensor.py
def media(valores):
    if not valores:
        return 0.0
    return sum(valores) / len(valores)

if __name__ == "__main__":
    leituras = [21.1, 22.4, 23.0, 21.8]
    print(f"Media Python: {media(leituras):.2f}")

File: Atividade 2/sensor.cpp
#include <iostream>
#include <vector>
using namespace std;

double media(const vector<double>& valores) {
    double soma = 0.0;
    for (double v : valores) soma += v;
    return valores.empty() ? 0.0 : soma / valores.size();
}

int main() {
    vector<double> leituras = {21.1, 22.4, 23.0, 21.8};
    cout << "Media C++: " << media(leituras) << endl;
    return 0;
}